neurons/miners/StableMiner/miner.py

```diff
@@ -28,24 +29,47 @@ def __init__(self):
 
     def load_models(self):
         # Load the text-to-image model
-        self.t2i_model = AutoPipelineForText2Image.from_pretrained(
-            self.config.miner.model,
+        self.t2i_model_custom = AutoPipelineForText2Image.from_pretrained(
+            self.config.miner.custom_model,
```

Comment:
Leave this as self.config.miner.model (the variable can be named model_custom though).